### PR TITLE
PR-specific build

### LINKED ISSUE #6: PR-specific travis CI
Ideally, we should only run the test the recipes that are changed in a pull request. building can take a long time, especially when large number of recipes are accumulated in the future.

### PR DESCRIPTION
**Pull request by RNAer**
This fixes #6 .

### CODE REVIEW

File: ci/main.py [hunk 1]
```diff
@@ -3,61 +3,119 @@
...
+    Returns
+    -------
+    boolean
+        If a previous package in the channel has the same name, the same
+        version, and an equal or higher build number, then return False;
+        otherwise, return True.
+    '''
+    cmd = ('conda search --json --override-channels '
+           '-c {0} --spec {1}').format(channel, name)
+    print('CHECK COMMAND: {0}'.format(cmd))
```
Comment (wasade) [inline ci/main.py]: These `print` statements almost look like they should be log statements via `logging`?

Comment (RNAer) [inline ci/main.py]: ok. i can replace it with `logging`.

**Comment by Jorge-C** [comment]
Hey, any reasons to not add documentation and logging on top of #12 which already solved 6?

**Comment by RNAer** [comment]
mainly because the low priority of windows?

**Comment by Jorge-C** [comment]
Sorry, that was a typo: I meant the pull request #12 from this afternoon.

**Comment by RNAer** [comment]
oh noooo. I didn't see that. sorry about that. We should have communicated better by assigning tasks... I can review you code or you can review my code? i am fine with either way.

**Comment by colinbrislawn** [comment]
haha wow. Talk about #12 No duplicate effort! :laughing:

**Comment by Jorge-C** [comment]
@RNAer I can merge both PRs to get an _enhanced_ master PR.

**Comment by RNAer** [comment]
:+1: go ahead!

**Comment by RNAer** [comment]
@Jorge-C , do you have an ETA?

**Comment by Jorge-C** [comment]
Later today, I'm planning on working on it after a meeting

**Pull request merged**